src/Compilers/CSharp/Portable/Emitter/Model/SynthesizedPrivateImplementationDetailsStaticConstructor.cs

```diff
@@ -7,10 +7,13 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
 {
     internal sealed class SynthesizedPrivateImplementationDetailsStaticConstructor : SynthesizedGlobalMethodSymbol
     {
+        private readonly PrivateImplementationDetails _details;
```

Comment:
It looks like base class already stores this information and exposes it via ContainingPrivateImplementationDetailsType property.